hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestS3BucketManager.java

```diff
@@ -101,31 +83,13 @@ public void testOzoneVolumeNameForUserFails() throws IOException {
 
   }
 
-  @Test
-  public void testDeleteS3Bucket() throws IOException {
```

Comment:
Same q. Why is this test not needed any more?

Reply:
Because S3BucketDeleteRequest is tested through TestS3BucketDeleteRequest.